stormforge/assets/dashboards/stormforge_overview.json

```
@@ -1,545 +1,1035 @@
 {
-  "title": "StormForge Optimize Live Application Overview",
-  "description": "View recommendations for configured applications",
+  "title": "StormForge Optimize Live Workload Overview",
+  "description": "[StormForge Optimize Live](https://www.stormforge.io/how-stormforge-optimize-live-works/) applies machine learning to your observability metrics to make real-time recommendations to resource requests for any deployment running in Kubernetes.\n\nWith StormForge Optimize Live, you can:\n\n* Improve resource efficiency\n* Leverage existing observability data\n* Reduce risk of performance issues\n* Achieve fast time to value\n* Deploy recommendations automatically or with approval\n\nThe Datadog Workload Overview dashboard lets you preview recommendations made by Optimize Live on a _single workload_ basis. To get started:\n\n1. Ensure Optimize Live is [installed](https://docs.stormforge.io/optimize-live/getting-started/install/) and [configured](https://docs.stormforge.io/optimize-live/getting-started/configuration/)\n2. Select a single value for each of the dashboard's template variables\n3. Review the Recommendation Summary group to understand the overall impact of the recommendation (inclusive of replica scale-up and scale-down events)\n4. Review the Details group to understand the impact of the recommendation on an average container (replica) in the workload\n5. When ready, use Optimize Live to start automatically applying the recommendations",
```

We typically like to have the overview in dashboards be a couple of sentences. Could you include a couple of sentences on the value of the dashboard?

---

I'll make and commit a suggested change here in the comment for ease of documentation. Let me know if this is better.

```suggestion
  "description": "StormForge Optimize Live applies machine learning to your observability metrics to make real-time recommendations to resource requests for any deployment running in Kubernetes. The Workload Overview dashboard lets you preview the recommendations made by Optimize Live, and the projected impact of applying those recommendations. The dashboard is designed to show recommendations and impact on a per-workload basis.",
```